lua/nvim-gps/init.lua

```diff
@@ -13,7 +13,12 @@ local default_config = {
 		["function-name"] = ' ',
 		["method-name"] = ' ',
 		["container-name"] = 'ﮅ ',
-		["tag-name"] = '炙'
+		["tag-name"] = '炙',
+		["hook-name"] = 'ﯠ ',
```

Comment:
Since all these capture types seem to be language specific, move them to language specific icons.

Reply:
Similarly for the other PR